python/ray/rllib/ppo/base_evaluator.py

```diff
@@ -12,24 +12,17 @@
 import numpy as np
 import ray
 
+from ray.rllib.optimizers import Evaluator, SampleBatch
 from ray.rllib.parallel import LocalSyncParallelOptimizer
 from ray.rllib.models import ModelCatalog
 from ray.rllib.envs import create_and_wrap
 from ray.rllib.utils.sampler import SyncSampler
 from ray.rllib.utils.filter import get_filter, MeanStdFilter
 from ray.rllib.utils.process_rollout import process_rollout
 from ray.rllib.ppo.loss import ProximalPolicyLoss
-from ray.rllib.optimizers import SampleBatch
 
 
-# TODO(pcm): Make sure that both observation_filter and reward_filter
-# are correctly handled, i.e. (a) the values are accumulated accross
-# workers (if necessary), (b) they are passed between all the methods
-# correctly and no default arguments are used, and (c) they are saved
-# as part of the checkpoint so training can resume properly.
-
-
-class Runner(object):
+class PPOEvaluator(Evaluator):
```

Comment:
extends TFMultiGPUSupport

Reply:
Would it be OK to do this in a later PR, where we implement `tf_loss_inputs` and `build_tf_loss` and introduce the `LocalMultiGPUOptimizer`?

Reply:
Sounds good, maybe leave a TODO